scripts/install-chef-suse.sh

```diff
@@ -612,6 +630,8 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
     # ubuntu admin node.
 fi
 
+echo "pre_sanity_checks, $(date)" >> $installation_steps
```

Comment:
I would prefer a space as a separator, easier to extract. Beside that the code smells like duplication, write a small func that just accepts the step :)